src/main/java/kubatech/api/helpers/GTHelper.java

```diff
@@ -52,4 +60,42 @@ public static int getVoltageTier(long voltage) {
     public static int getVoltageTier(GT_MetaTileEntity_MultiBlockBase mte) {
         return (int) getVoltageTierD(mte);
     }
+
+    public static class StackableGUISlot {
+
+        public StackableGUISlot() {};
+
+        public StackableGUISlot(int count, ItemStack stack, ArrayList<Integer> realSlots) {
+            this.count = count;
+            this.stack = stack;
+            this.realSlots = realSlots;
+        }
+
+        public int count;
+        public ItemStack stack;
+        public ArrayList<Integer> realSlots = new ArrayList<>();
+
+        public void write(PacketBuffer buffer) throws IOException {
+            buffer.writeVarIntToBuffer(count);
+            buffer.writeItemStackToBuffer(stack);
+        }
+
+        public static StackableGUISlot read(PacketBuffer buffer) throws IOException {
+            StackableGUISlot slot = new StackableGUISlot();
+            slot.count = buffer.readVarIntFromBuffer();
+            slot.stack = buffer.readItemStackFromBuffer();
+            return slot;
```

Comment:
```suggestion
            return new StackableGUISlot(
                buffer.readVarIntFromBuffer(),
                buffer.readItemStackFromBuffer(),
                new ArrayList<>());
```
Then `final` can be applied to fields.